target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -133,6 +133,7 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:wan" "3:lan" "4:lan"
 		;;
+	buffalo,wzr-hp-g300nh|\
 	dlink,dir-825-b1)
```

Comment:
Please move the block to maintain alphabetic sorting in the file.

Reply:
Umm move where? This has similar configuration with dir-825-b1. The file is sorted so that devices with similar configuration are grouped together and sorted alphabetically. I cannot see where else to put this.

Reply:
The whole file is sorted based on the first device in each case. So, move the whole block
```
	buffalo,wzr-hp-g300nh|\
	dlink,dir-825-b1)
		ucidef_set_interface_wan "eth1"
		ucidef_add_switch "switch0" \
			"0:lan" "1:lan" "2:lan" "3:lan" "5@eth0"
		;;
```
up so comes before buffalo,wzr-hp-g302h-a1a0)